packages/lib/src/vis/matrix/MatrixVis.module.css

```diff
@@ -74,6 +74,8 @@
   align-items: center;
   justify-content: center;
   background-color: var(--h5w-matrix-cell--bgColor, transparent);
+  overflow: hidden;
+  white-space: nowrap;
```

Comment:
Added this to prevent text wrapping and overflow like on the screenshot in #916  (note that `text-overflow: ellipsis` does not work due to the use of Flexbox, and I don't think it's wise to add an extra `span` in every cell.)